code/game/objects/machinery/squad_supply/supply_console.dm

```diff
@@ -34,9 +35,15 @@
 			supply_pad = _supply_pad
 			return
 
+
+/obj/machinery/computer/supplydrop_console/proc/ping_beacon()
+	SIGNAL_HANDLER
+	playsound(src,'sound/machines/terminal_prompt_confirm.ogg', 50, TRUE)
+
 /obj/machinery/computer/supplydrop_console/Destroy()
 	supply_beacon = null
 	supply_pad = null
+	UnregisterSignal(SSdcs, COMSIG_GLOB_SUPPLY_BEACON_CREATED)
```

Comment:
sigs are already unregistered on destroy, you don't need to specify it.

Reply:
Oh, neat. The AI has a lot of those global unregisters. Should I nuke them?

Reply:
yes, but not in this pr

Reply:
Done

Reply:
uh, you still need to remove this.